### PR TITLE
Fix problematic case with opts==None and therefore no opts.use_dropout.

### PR DESCRIPTION
**Pull request by benelot**
I found some problem(s) while reading through your code and running it with different settings. I think it would be helpful if I contributed some fix(es) back while I do this. For now, there is only one fix in the request but maybe there are more to come.

### CODE REVIEW

File: event_log.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-import event_pb2
+from tensorflow.core.util import event_pb2
```
Comment (matpalm) [inline event_log.py:2]: this is actually importing a different proto; i'm wanting to use the `event.proto` at the root of this repro.

it's a bit hacky as it is now; this code requires you run `protoc event.proto --python_out=.` (as mentioned in the README.md) before the `event_pb2.py` is created. what i should do is try the import and, if it fails, give a warning that this code needs to be run (either that or make it a proper part of some kind of `setup.py` init...

**Review by matpalm — COMMENTED** [review]
thanks for the pull request. see note re: the event_pb, i should have made this code more robust and explicit about the requirement to compile the proto file sorry...

**Comment by benelot** [comment]
Oh ok, I will check later if I can add this quickly to the setup.py if you do not have time to fix it quickly. It seems to work with the other one as well btw, but I did not look into the details, I just ran it and it trained. So I can not say if it actually works or if the code just "runs".

**Comment by matpalm** [comment]
yeah the import would have worked but then it would have failed if you'd being using any type of event logging (i.e. `--event-log=...`)  i use the logging of runs for debugging, prepopulating replay memories etc, but it's not a requirement, you can train / run without it (which is why i think i'll move the import to just before it's being used) thanks for the sanity checks!

**Comment by benelot** [comment]
Thanks, nice to know!

**Pull request closed without merging**